openraft/src/core/admin.rs

```diff
@@ -315,7 +315,7 @@ impl<'a, D: AppData, R: AppDataResponse, N: RaftNetwork<D>, S: RaftStorage<D, R>
     /// Return true if removed.
     #[tracing::instrument(level = "trace", skip(self))]
     pub fn try_remove_replication(&mut self, target: NodeId) -> bool {
-        tracing::debug!(target, "try_remove_replication");
+        tracing::debug!(target = display(target), "try_remove_replication");
```

Comment:
Great! `display(target)` looks better than `%target`!

Reply:
> Great! display(target) looks better than %target!

Well, to be honest, I didn't look into `tracing` framework in detail yet. We are using internally a different beast :-).

Reply:
So do I guess:D